docs/blog/2018-03-07-why-we-created-the-plugin-library/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Why we created the Gatsby plugin library
+date: 03-07-2018
+author: Shannon Soper
+---
+
+## Why build a plugin / package library
+
+In over 60 interviews with Gatsby users, plugin discoverability appeared as a top pain point (see [blog post on the Gatsby UX research program](/2017-12-20-introducing-the-gatsby-ux-research-program/) for a full list of pain points). So we built a plugin / package library [here](/packages/)!
+
+In this article, I want to explain how UX research drove the development and design of this plugin library and invite you all to follow and contribute as the library continues to improve.
+
+
+## Jobs to be done
+
+After talking to over 60 Gatsby users, I compiled what’s called an “empathy map” in UX design to put myself in the shoes of Gatsby users; basically, this map describes a summary of what the typical Gatsby user experiences when searching for plugins.
+
+![Empathy map](plugin-map.png)
+
+Based on the empathy map above, we identified five “jobs” that Gatsby users want done regarding plugins and packages:
+* Make it easy to search for and find the best, most recent, and relevant plugin for their site.
```

Comment:
Can we be more succinct here? Jobs to be done should be short and snappy and should not be feature lists. To use Clayton Christiansen's famous example, a job to be done is "entertain myself on a boring drive", the product is a milkshake, the feature is random chunks in the milkshake. It seems like you're sub-bulleting features under jobs to be done, would suggest to remove those sub-bullets.

Here the job to be done is "Find relevant, high-quality plugins for their site."

Reply:
done